### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by mweel1** [rewindui/rewindui#49] [General]
How do you imagine dark mode integrating with this framework?

**Comment by NickDunas** [comment]
I will just add the dark mode classes on each component. Then it's up to each developer which strategy they want to follow on their project.

For the documentation I plan to go with supporting system preference and manual selection to showcase the dark mode styles.

**Reply by fortezhuo** [reply]
Hi, do you provide some example for dark mode integration ?

**Reply by NickDunas** [reply]
Dark mode is not yet implemented @fortezhuo. It will be implemented once I finish with the Sidebar component and full test coverage!
